src/docks/playlistdock.ui

```diff
@@ -59,14 +59,21 @@
            <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
 &lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
 p, li { white-space: pre-wrap; }
-&lt;/style&gt;&lt;/head&gt;&lt;body&gt;
+&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
```

Comment:
Also when editing a UI XML file with rich text like this you must revert changes like this that add font information. This font is not cross platform, and this change should not be included.

Reply:
Ok I will revert these changes. I am not sure how they were added. I only edited this file using Qt designer, that's why I didn't check it when committing. Are you able to give an idea about what happened so that I avoid it in the future.

I will revert the file to before the commit.

Reply:
That is the behavior of Qt Designer whether inside Qt Creator or not. There is no way to avoid it. You just need to review what you are committing.